NextCenturyCorporation/itm-evaluation-client
# Itm 86: Add gradle config for swagger generation

To test:
Manually delete the swagger_client/api and swagger_client/models folders.
refresh your IDE or do a "git status" on the command line to see that git see the files as missing

run "./gradlew" to run the swagger generation

refresh your IDE or do a "git status" again. If all the files are generated correctly there should be no changes from git's perspective.
